packages/cli/src/commands/deploy.ts

```diff
@@ -155,6 +155,9 @@ export default class Deploy extends AuthCommand {
       }
       output.push('')
     }
+    if (!creating.length && !deleting.length && !updating.length) {
```

Comment:
I think you could move this `if` just after line `131` and, instead of `output.push(<message>)` just `return <message>`.

Reply:
Good point, that makes sense